### PR TITLE
move stable iotcentral Go SDK package out of preview directory

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
The 2018-09-01 package was incorrectly placed under the preview
directory.  Moved Go SDK config section to its own config file.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/iotcentral/resource-manager/readme.go.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+### Tag: package-2018-09-01 and go
+
+These settings apply only when `--tag=package-2018-09-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2018-09-01' && $(go)
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2018-09-01/$(namespace)
+```
+
+### Tag: package-2017-07-01-privatepreview and go
```
Comment (vladbarosan) [inline specification/iotcentral/resource-manager/readme.go.md]: really ? should we even leave that in ?

Comment (vladbarosan) [reply specification/iotcentral/resource-manager/readme.go.md]: im talking about hte privatepreview api

**Review by vladbarosan — APPROVED** [review]
LGTM

File: specification/iotcentral/resource-manager/readme.go.md (hunk 1 above)
Comment (jhendrixMSFT) [reply specification/iotcentral/resource-manager/readme.go.md]: @emgarten do we need to keep building this API version?

**Review by emgarten — APPROVED** [review]
Thanks for fixing this @jhendrixMSFT  🍰

**Pull request merged**